app/views/main.scala.html

```diff
@@ -59,6 +59,32 @@
         <div class="govuk-width-container ">
             <div class="govuk-footer__meta">
                 <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+                    <ul class="govuk-footer__inline-list">
+                        <li class="govuk-footer__inline-list-item">
+                          <a class="govuk-footer__link" href="@routes.HelpGuideController.helpGuide()">
+                            Help
+                          </a>
+                        </li>
+                        <li class="govuk-footer__inline-list-item">
+                          <a class="govuk-footer__link" href="#">
+                            Contact
+                          </a>
+                        </li>
+                        <li class="govuk-footer__inline-list-item">
+                          <a class="govuk-footer__link" href="@routes.AccessibilityController.accessibilityStatement()">
+                            Accessibility statement
+                          </a>
+                        </li>
+                        <li class="govuk-footer__inline-list-item">
+                          <a class="govuk-footer__link" href="">
+                            Cookies
```

Comment:
I think we came to the conclusion that we don't need a cookies page for now because we're only using essential cookies for auth - we don't have any analytics cookies that would need a cookie page. Is that right? Can we remove this link?

Reply:
Yes definitely!